backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/computing/purchasecontract/AbstractInterestComputingMethod.java

```diff
@@ -129,6 +130,11 @@ public boolean applies(final @NonNull TableRecordReference recordRef, @NonNull f
 			return contractProvider.streamModularPurchaseContractsForInvoiceLine(InvoiceLineId.ofRepoId(recordRef.getRecord_ID()));
 		}
 
+		if (recordRef.tableNameEqualsTo(I_C_Flatrate_Term.Table_Name))
```

Comment:
Now that both AV and SV computing methods implement their own applies, and I don't see a verbose call to super, is this method still needed?

Reply:
I prefer to leave it here since the 2 implementations will have 2 of the 3 options.